### PR TITLE
Addresses menu-scroll-down issue

### LINKED ISSUE #280: menu-scroll-down scrolls too far
The menu scroll down scrolls passed the beginning of the content making it hard to read. It's set to the class '#primary', but maybe we can make it so it scrolls to where the bottom of the nav meets the top of the content.

demo

### PR DESCRIPTION
**Pull request by joshcummingsdesign**
Addresses scroll issue in #280

### CODE REVIEW

**Comment by melchoyce** [comment]
Looks like this might be a duplicate of #274?

**Comment by joshcummingsdesign** [comment]
Indeed it is, updated and close duplicate issue.

**Comment by joshcummingsdesign** [comment]
@melchoyce Yes, this addresses the same issue.

File: assets/js/global.js [hunk 1]
```diff
@@ -40,28 +40,27 @@ jQuery( document ).ready( function( $ ) {
...
-		$navigationHeight = $navigation.outerHeight();
-
 		if ( 'none' === $( '.menu-toggle' ).css( 'display' ) ) {
 			$branding.css( 'margin-bottom', $navigationHeight );
 		} else {
 			$branding.css( 'margin-bottom', '0' );
 		}
 	}
 
+	function getNavProps() {
```
Comment (tywayne) [inline assets/js/global.js:71]: I would suggest changing this function name. A 'get' in the function name implies the function will return something. Maybe `setNavProps()`?

**Comment by tywayne** [comment]
Tested this PR, and it appears the offset is different when wpadminbar is present, than when it isn't. We should adjust that offset accordingly.

**Comment by melchoyce** [comment]
Closing this as a duplicate of #274.

**Pull request closed without merging**